packages/fluentui/react-northstar/src/components/Popup/Popup.tsx

```diff
@@ -540,7 +545,7 @@ const Popup: React.FC<PopupProps> &
   const element = (
     <>
       {triggerElement}
-      {open && (inline ? contentElement : <PortalInner mountNode={mountNode}>{contentElement}</PortalInner>)}
+      {inline ? contentElement : <PortalInner mountNode={mountNode}>{contentElement}</PortalInner>}
```

Comment:
The Animation component is responsible for mounting/removing the element now, so no need for this check